### PR TITLE
CASMINST-5258 Linting/formatting

### PR DESCRIPTION
**Pull request by rustydb**
# Description

Various linting with some opinionated formatting for the new/alpha "Add NCN Data page."

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md [hunk 1]
```diff
@@ -30,240 +42,288 @@ Scenarios where this procedure is applicable:
...
+#### Saving/Reloading
 
-            Example output:
+1. (`ncn-mw#`) Save the currently collected values to a loadable script for reloading.
 
-            ```text
-            10.254.1.26
-            ```
+    ```bash
+    cat << EOF > ${NODE}-env-vars.sh
```
Comment (rustydb) [inline operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:214]: Going to update this to make the file in either `/metal/recovery`, the s3fs_cache, or the CSM directory in /usr or /opt.

File: operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md [hunk 2]
```diff
@@ -422,38 +489,38 @@ Scenarios where this procedure is applicable:
...
+    > 1. (`ncn-mw#`) Verify that the BMC is reachable at the expected IP address.
+    >
+    >    ```bash
+    >    ping "${NODE}-mgmt"
+    >    ```
+    >
+    > 1. (`ncn-mw#`) Verify that the BMC Redfish `v1/Managers` endpoint is reachable.
+    >
+    >    ```curl
+    >    curl -f -u root:$IPMI_PASSWORD https://${BMC_XNAME}/redfish/v1/Managers
```
Comment (rustydb) [inline operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:522]: Double-checking this works, I can't remember if I needed the `-k` for working around the untrusted https cert.

Think I meant to append `-f` not replace `-k` with it.

Comment (rsjostrand-hpe) [reply operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:522]: You need to specify `-k` as we are not currently rolling out certificates generated by the System to BMCs yet.

**Review by rsjostrand-hpe — APPROVED** [review]

File: operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md (hunk 2 above)
Comment (rsjostrand-hpe) [reply operations/node_management/Add_Remove_Replace_NCNs/Add_NCN_Data.md:522]: You need to specify -k as we are not currently rolling out certificates generated by the System to BMCs yet.

**Review by rsjostrand-hpe — CHANGES_REQUESTED** [review]

**Review by rsjostrand-hpe — APPROVED** [review]

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**